src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCFIntegrationTest.java

```diff
@@ -578,6 +578,32 @@ public void testFilters() throws IOException {
         Assert.assertEquals(filteredRefBlockVC.getGenotype(0).getDP(), 12); // Ref block is combination of filtered variant with depth 22 and filtered ref block with depth 1
     }
 
+    @Test
+    public void testMovingFilters() throws IOException {
+        final File input = getTestFile("dragen.g.vcf");
+        final File output = createTempFile("reblockedgvcf", ".vcf");
+
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addReference(new File(hg38Reference))
+                .add("V", input)
+                .add("L", "chr1")
+                .add(ReblockGVCF.ADD_FILTERS_TO_GENOTYPE, true)
+                .addOutput(output);
+        runCommandLine(args);
+
+        final VariantContext filteredVC = VariantContextTestUtils.readEntireVCFIntoMemory(output.getAbsolutePath()).getRight().get(3); // last site in the file
+        Assert.assertFalse(filteredVC.isFiltered());
+        Assert.assertTrue(filteredVC.getGenotype(0).isFiltered());
+
+        final VariantContext unfilteredVC = VariantContextTestUtils.readEntireVCFIntoMemory(output.getAbsolutePath()).getRight().get(1);
+        Assert.assertFalse(unfilteredVC.isFiltered());
+        Assert.assertFalse(unfilteredVC.getGenotype(0).isFiltered());
+
+        final VariantContext filteredRefBlockVC = VariantContextTestUtils.readEntireVCFIntoMemory(output.getAbsolutePath()).getRight().get(0);
+        Assert.assertFalse(filteredRefBlockVC.isFiltered()); // Ref block is unfiltered even though the input RefBlock and low qual variant were both filtered
+        Assert.assertFalse(filteredRefBlockVC.getGenotype(0).isFiltered()); // Ref block genotype is also unfiltered
```

Comment:
Hmm, that's a good question. I had assumed that it didn't make sense to have a ref block with a genotype level filter. The low quality variant here was dropped (incorporated into the Ref Block), and I was thinking that the GQ was enough information here that it would only be more confusing if we included the filter in the ref block. 

I suppose the outcome of this is that if we kept the filter status in the ref block it would make it into the final VCF at any sites that overlap the entire ref block. So if one variant had a dragen hard filter applied, but then was incorporated into a RefBlock, then all of the `0/0` genotypes would be filtered across the entire ref block, not just the original filtered site. I think this would be more confusing than it's worth, but happy to hear if you think we want to propagate that filter through.

Reply:
To follow up here: we spoke offline and the conclusion was that GQ0 basically acts as a filter for ref blocks so it's ok to not propagate the filter from the low quality variants.